web/api/web_api_v1.c

```diff
@@ -697,7 +710,7 @@ static struct api_command {
         { "alarm_log",       0, WEB_CLIENT_ACL_DASHBOARD, web_client_api_request_v1_alarm_log       },
         { "alarm_variables", 0, WEB_CLIENT_ACL_DASHBOARD, web_client_api_request_v1_alarm_variables },
         { "allmetrics",      0, WEB_CLIENT_ACL_DASHBOARD, web_client_api_request_v1_allmetrics      },
-
+        { "manage/health",   0, WEB_CLIENT_ACL_MGMT,      web_client_api_request_v1_mgmt_health     },
```

Comment:
I think this does not work.
The tokenizer splits words at `/` so only `manage` will be visible. Are you sure this works?

Generally, you should provide a function for `manage` and then parse the various `manage` functions there.

Reply:
It works, it's tested. I will refactor if needed, when a second mgmt api is added.